src/test/adapter.ts

```diff
@@ -9,7 +9,7 @@ const assert = chai.assert;
 
 describe('PHP Debug Adapter', () => {
 
-    const TEST_PROJECT = path.normalize(__dirname + '/../../testproject');
+    const TEST_PROJECT = path.normalize(__dirname + '/../../testproject/main');
```

Comment:
when testing development of this feature, I had to test that multiple paths were working, so I created multiple paths in `testproject`. I thought that in the future other developers would also need to test multiple paths, but if you'd like I can move the files back.